python/py-url-normalize/Portfile

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-url-normalize
+version             1.4.3
+revision            0
+categories-append   devel net
+supported_archs     noarch
+license             LGPLv2+
```

Comment:
this isn't not correct as also pointed out by `port lint`, it should be `LGPL-2+` instead. As a side note, to make sure that you "lint" everything and, for example, make sure that dependencies are available for all Pyhon versions, you should do `port lint --nitpick subportof:py-<portname>`.

Reply:
Fixed with the latest push